cinema/urls.py

```diff
@@ -10,14 +11,19 @@
     OrderViewSet,
 )
 
-router = routers.DefaultRouter()
-router.register("genres", GenreViewSet)
-router.register("actors", ActorViewSet)
-router.register("cinema_halls", CinemaHallViewSet)
-router.register("movies", MovieViewSet)
-router.register("movie_sessions", MovieSessionViewSet)
-router.register("orders", OrderViewSet)
+#router = routers.DefaultRouter()
+actors = ActorViewSet.as_view({'get': 'list', 'post': 'create'})
+# router.register("genres", GenreViewSet)
+# router.register("movies", MovieViewSet)
+# router.register("cinema_halls", CinemaHallViewSet)
+# router.register("movies", MovieViewSet)
+# router.register("movie_sessions", MovieSessionViewSet)
+# router.register("orders", OrderViewSet)
```

Comment:
Consider uncommenting this line if you want to register the `OrderViewSet` with the router.